deploy/crds/pulpproject_v1beta1_pulp_crd.yaml

```diff
@@ -150,12 +150,30 @@ spec:
                   - ingress
                   - Route
                   - route
+                  - LoadBalancer
+                  - loadbalancer
+                  - NodePort
+                  - nodeport
               ingress_annotations:
                 description: Annotations to add to the ingress
                 type: string
               ingress_tls_secret:
                 description: Secret where the ingress TLS secret can be found
                 type: string
+              loadbalancer_annotations:
+                description: Annotations to add to the loadbalancer
+                type: string
+              loadbalancer_protocol:
+                description: Protocol to use for the loadbalancer
+                type: string
+                default: http
```

Comment:
how hard is making `https` the default?

Reply:
I believe it has more to do with the setup of the external load balancer and how its configured.